src/main/java/edu/cornell/kfs/sys/service/CuBatchFileDirectoryService.java

```diff
@@ -1,12 +1,13 @@
 package edu.cornell.kfs.sys.service;
 
+import org.kuali.kfs.sys.batch.service.BatchFileDirectoryService;
 import org.kuali.kfs.sys.businessobject.SystemOptions;
 import org.kuali.rice.core.api.util.KeyValue;
 import org.springframework.cache.annotation.Cacheable;
 
 import java.util.List;
 
-public interface BatchFileDirectoryService {
+public interface CuBatchFileDirectoryService {
```

Comment:
Why are we changing of name of this class in the KFS upgrade user story?

Reply:
With the 06/21 financials patch, KualiCo added their own implementation of BatchFileDirectoryService. I believe our version of the service was renamed to avoid confusion, and to simplify the setup of our custom service's implementation.